storage/client_test.go

```diff
@@ -37,7 +37,7 @@ func TestCreateBucketEmulated(t *testing.T) {
 		}
 		got, err := client.CreateBucket(context.Background(), project, want)
 		if err != nil {
-			t.Fatal(err)
+			t.Fatalf("%s, %v", transport, err)
```

Comment:
Maybe this is a good case for using subtests (`t.Run()`)? Same for the bucket tests below -  then you can separate each transport

Reply:
Yeah we can do that! Can I do that in a separate PR because there is an existing test in here (and in some other local branches I have) I'd like to update at once.

Reply:
Sounds good 👍